cmd/registry/cmd/export/yaml.go

```diff
@@ -35,43 +38,39 @@ func yamlCommand(ctx context.Context) *cobra.Command {
 				log.FromContext(ctx).WithError(err).Fatal("Failed to get client")
 			}
 
-			adminClient, err := connection.NewAdminClient(ctx)
-			if err != nil {
-				log.FromContext(ctx).WithError(err).Fatal("Failed to get client")
-			}
-
 			var name string
 			if len(args) > 0 {
 				name = args[0]
 			}
 
-			if project, err := names.ParseProject(name); err == nil {
-				_, err := core.GetProject(ctx, adminClient, project, func(message *rpc.Project) {
-					core.ExportYAMLForProject(ctx, client, adminClient, message)
-				})
+			if _, err := names.ParseProject(name); err == nil {
+				patch.ExportProject(ctx, client, name)
 				if err != nil {
 					log.FromContext(ctx).WithError(err).Fatal("Failed to export project YAML")
 				}
 			} else if api, err := names.ParseApi(name); err == nil {
 				_, err = core.GetAPI(ctx, client, api, func(message *rpc.Api) {
-					core.ExportYAMLForAPI(ctx, client, message)
+					bytes, _, err := patch.ExportAPI(ctx, client, message)
+					if err != nil {
+						log.FromContext(ctx).WithError(err).Fatal("Failed to export API")
+					} else {
```

Comment:
Nit: Prefer to keep the normal flow of the program at the same level of indentation. https://github.com/golang/go/wiki/CodeReviewComments#indent-error-flow

Reply:
Done